### ISSUE TITLE
NUXT support

### ISSUE THREAD

**Issue by jakewhiteley** [darkroomengineering/tempus#9]
Hi guys,

Could you consider delaying assignment of `originalRAF`/`originalCancelRAF` until `patch` is called so that I can use this in Nuxt without getting any window is undefined errors before hydration?

**Comment by clementroche** [comment]
Indeed, I figured it out while working on react implementation. Also, you may face another issue because Tempus will be `undefined` if executed on the server side. So perhaps it should just be inert instead of `undefined`

please try [v1.0.0-dev.5](https://www.npmjs.com/package/@darkroom.engineering/tempus)

**Comment by jakewhiteley** [comment]
@clementroche `npm error notarget No matching version found for @darkroom.engineering/tempus@1.0.0-dev.5.`

I also notice that `npm i @darkroom.engineering/tempus` doesn't work on windows, 
and you have to use `npm i "@darkroom.engineering/tempus"`. I assume this is due to the period in your organisation name.

**Comment by clementroche** [comment]
Sorry I linked to the wrong package. Please use [tempus@1.0.0-dev.5](https://www.npmjs.com/package/tempus) instead
